tests/js/snapshots.test.js

```diff
@@ -156,10 +156,21 @@ test.describe("Furo menu bars", () => {
   });
 });
 
-test("left side bar table of contents renders correctly", async ({ page }) => {
-  await page.goto("");
-  const leftToC = page.locator("div.sidebar-tree");
-  await expect(leftToC).toHaveScreenshot();
+test.describe("left side bar", () => {
+  test("table of contents renders correctly", async ({ page }) => {
+    await page.goto("");
+    const leftToC = page.locator("div.sidebar-tree");
+    await expect(leftToC).toHaveScreenshot();
```

Comment:
would it be possible to expand this screenshot of the sidebar to include the languages component? Then we can check that it renders correctly as part of the full sidebar.

Reply:
also then you can probably remove the first screenshot of the 2 in the "translations render correctly" test so we're not duplicating

Reply:
That's a great idea! Then we capture the spacing between each distinct sidebar element. Thanks!